### PR TITLE
Add Custom PC AMD Ryzen

### PR DESCRIPTION
**Pull request by dangthaison91**
Custom PC - Mac OS 10.15.4:

CPU: AMD Ryzen 9 3950X 16-Cores @4.3GHz All Cores
RAM: 32 GB 3600MHz
SSD: Samsung 970 Evo Plus 500GB MZ-V7S500BW
Motherboard: ASRock X570 Taichi
GPU: Sapphire Nitro+ Radeon RX 5700XT SE

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Xcode 11
 -------
 |🖥 | Computer Model | CPU | RAM | Fresh Build Time | Incremental Build Time | Xcode Version | Date & Commit Hash | New Build System? |
 |-- | -------------- | --- | --- | ---------------- | ---------------------- | ---- | ---- | ----------------- |
+|⌨️ | [Custom PC](https://github.com/ashfurrow/xcode-hardware-performance/pull/108)| AMD Ryzen 9 3950X 4.3GHz(all cores) | 32 GB | 0:21 | 0:03 | 11.4 | 2020-04-16 ([commit](https://github.com/dangthaison91/xcode-hardware-performance/commit/96f9eda856c171fccbb159bc7c4e8ced720abc31) | :heavy_check_mark:|
```
Comment (ashfurrow) [inline Readme.md]: This GitHub link should be to the Eidolon repository, the commit that you used.

**Review by ashfurrow — COMMENTED** [review]
Looks good! Just one thing to clarify and then we can merge. Thanks for your submission!

File: Readme.md (hunk 1 above)
Comment (dangthaison91) [reply Readme.md]: I changed it to PR link:
https://github.com/ashfurrow/xcode-hardware-performance/pull/126

**Review by ashfurrow — APPROVED** [review]
Fab, thanks again!

File: Readme.md (hunk 1 above)
Comment (thii) [reply Readme.md]: Shouldn't it be a commit of https://github.com/artsy/eidolon? Also, Custom PC is pointing to a different PR: https://github.com/ashfurrow/xcode-hardware-performance/pull/108

Comment (ashfurrow) [reply Readme.md]: Yeah, that's a good point. We should have the readme point to the commit _and_ this PR (see other examples). @dangthaison91 I assume you were using the latest commit from `master` at the time? I can update the readme if you give the :+1:

Comment (dangthaison91) [reply Readme.md]: Oops, good catch. @ashfurrow Please, go on :D

Comment (ashfurrow) [reply Readme.md]: Awesome, thanks! All done 👍

**Comment by ergunkocak** [comment]
Hi @dangthaison91 ,

Thanks very much for sharing HW details. If you can make a blog post or video explaining how to make this awesome thing that would be even more awesome. I am looking into new HW and this PC just stole my heart :D 

Best

**Comment by ergunkocak** [comment]
Also sharing tips and tricks on how to make xcode work on it, will be really appreciated.

**Pull request merged**